common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -48,12 +48,11 @@ platform_init() {
     control_ip=192.168.250.140
     
     ip link add @BEEROCKS_BRIDGE_IFACE@ address "${base_mac}:00:00" type bridge
-    ip link add @BEEROCKS_BH_WIRE_IFACE@ type dummy
     ip link add wlan0 address "${base_mac}:00:10" type dummy
     ip link add wlan2 address "${base_mac}:00:20" type dummy
     ip link set dev wlan0 up
     ip link set dev wlan2 up
-    for iface in wlan0 wlan2 @BEEROCKS_BH_WIRE_IFACE@ $DATA_IFACE
+    for iface in wlan0 wlan2 $DATA_IFACE
```

Comment:
I don't understand this one. Isn't it the purpose of the platform_init code to make (for Linux target) a bridge and put wireless and wired interfaces under similarly to a typical AP?

Reply:
The DATA_IFACE is supposed to be the wired backhaul interface. I also tried to explain that in the commit message.

Reply:
Ah, I didn't read the commit messages, just the review. No more references to BEEROCKS_BH_WIRE_IFACE in the script and passing the interface name in as a parameter sounds good.